api/handlers.go

```diff
@@ -39,16 +40,31 @@ type (
 		Storage           int64  `json:"storageLimit"`
 	}
 
-	// userUpdateData defines the fields of the User record that can be changed
+	// accountRecoveryDTO defines the payload we expect when a user is trying to
+	// change their password.
+	accountRecoveryDTO struct {
+		Token           string `json:"token"`
+		Password        string `json:"password"`
+		ConfirmPassword string `json:"confirmPassword"`
```

Comment:
Is this the "repeat your password" field you usually have in forms? Hard to tell from the diff without docstrings but if it is that feels like something that should be handled by some JS on the website.

Reply:
Yes, this is something that I though as well, and Karol mentioned, too. The main reason it's here is to make sure we don't forget that this part of the process needs to be handled. Karol and I agreed that it should be handled by the frontend but left it like this with "we'll fix this later".

I'd rather F/U on this than address it here and break the portal code we have. It's on my radar.

Reply:
https://github.com/SkynetLabs/skynet-accounts/issues/111